hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/store/contracts/precompile/HTSTestsUtil.java

```diff
@@ -504,6 +504,26 @@ public class HTSTestsUtil {
                             .build(),
                     payer));
 
+    public static final List<BalanceChange> balanceChangesForLazyCreateNFT = List.of(
```

Comment:
I don't get it.  Between here and the preceeding `balanceChangesForLazyCreateHappyPath` the only thing differences are the two token serial numbers.  So why is it needed?

Reply:
After the changes in `TransferPrecompile`  the `TransferPrecompilesTest.lazyCreateExceedingResourceLimitThrows` and `TransferPrecompilesTest.nftTransferToLazyCreateHappyPathWorks` tests interfere with each others because of the common nft's lazy created account receiver.